src/Arachne.h

```diff
@@ -342,12 +345,15 @@ struct ThreadContext {
     /// context shall wait on this CV.
     ConditionVariable joinCV;
 
-    // Unique identifier for the core that this thread currently lives on.
-    // Used to index into global arrays with information per core.
+    /// Unique identifier for the core that this thread currently lives on.
+    /// Used to index into global arrays with information per core.
     /// This will only change if a ThreadContext is migrated when scaling down
     /// the number of cores.
     uint8_t coreId;
 
+    /// Thread class of this thread, used by the core policy.
```

Comment:
Comment mostly duplicates the code. You could shorten to:

/// Passed to the CorePolicy to choose core(s) for execution.

Let's discuss.